packages/javascript-sdk/src/http-client/index.ts

```diff
@@ -120,12 +120,8 @@ abstract class HttpClient extends Dispatcher {
         // Walk through auth tree
         await this.stepIterator(initialStep, options.authorization.handleStep, type, tree);
         // See if OAuth tokens are being used
-        let tokens;
-        try {
-          tokens = await TokenStorage.get();
-        } catch (err) {
-          // No OAuth Tokens
-        }
+        const tokens = await TokenStorage.get();
```

Comment:
Now that calling this method without tokens doesn't throw, we no longer need to wrap everything in try-catches.